README.md

```diff
@@ -10,19 +10,18 @@ A library for parsing and querying CSV data with Apache Spark, for Spark SQL and
 This library requires Spark 1.3+
 
 ## Linking
-You can link against this library in your program at the following coordiates:
+You can link against this library in your program at the following coordinates:
 
-### Scala 2.10
 ```
 groupId: com.databricks
-artifactId: spark-csv_2.10
+artifactId: spark-csv_2.11 or spark-csv_2.10
 version: 1.3.0
 ```
-### Scala 2.11
+
+For example, if you use `sbt` and scala 2.11. You would add a line like
+
 ```
-groupId: com.databricks
```

Comment:
Please do not remove this part. You can add the SBT version (and maybe Maven as well) next to it.